contents/product-engineers/nps-vs-csat-vs-ces.md

```diff
@@ -0,0 +1,202 @@
+- **Before subscription renewal –** to allow you to address any issues beforehand.
+- **After customer support interaction –** to see if customers feel their issue has been resolved.
+
+## What is CES?
+
+Customer effort score (CES) measures how easy it is to use your product or feature. 
+
+Users are asked "How easy was it to use `<name of feature>`" and a response scale ranging from 1 (very difficult) to 5 (very easy).
+
+![CES survey](../images/blog/nps-csat-ces/ces-survey.png)
+
+CES is great for debugging UX problems in your app and pinpointing problem areas.
+
+### How to calculate CES score
+
+To find your CES score, you calculate the average score of all your responses. That is: `sum of all responses / number of responses = CES score`
+
+![CES score](../images/blog/nps-csat-ces/ces-score.png)
+
+For example, let's say you survey `100` people. If the sum of all their responses is `450`, your CES score is `4.5`.
+
+### What is a good CES score for SaaS?
+
+While there are no published benchmarks for CES scores, you should aim for a score of 4 or higher. Any lower is a sign that users are struggling to use your app and this will lead to churn.
+
+### Drawbacks of CES
+
+CES surveys focus on a single feature interaction and are very narrow in scope. So, while they help you debug UX issues, they don't give you insights into overall satisfaction with your feature.
+
+Additionally, CES only tells you if a feature is difficult to use, but it doesn't tell you _why_. To counteract this, you need to ask follow-up questions, such as:
+
+- What makes `<feature name>` difficult to use?
+- What could we do to make it easier to use?
+
+### When to use CES surveys
+
+To best debug UX problems, you should use CES surveys after key interactions such as:
+
+- **After using a feature for the time** – to understand how intuitive it is.
+- **After a major UI update** – to confirm users can still navigate through your app effectively.
+- **When a user is migrating from a competitor's product to yours** – to highlight areas for improvement in onboarding.
+
+## NPS vs CSAT vs CES: Which one is best?
+
+In an ideal world, you could use all three survey types. However, survey fatigue means that it's not possible to send surveys to your users constantly, so you need to prioritize which ones to use.
+
+Our recommendation is to use CSAT before you have [product-market fit](https://posthog.com/founders/measure-product-market-fit), and NPS after you do:
+
+Before product-market fit, your app is constantly changing, so it makes sense to focus on the short-term only and thus use CSAT. Conversely, once you've achieved product-market fit, you can focus on the long-term and use NPS.
+
+Lastly, we prefer to avoid using CES as its scope is too narrow. There are other ways to understand if a feature is easy to use, like interviews or user testing. CSAT or NPS surveys generate far better insights, so we prefer to avoid fatiguing users with CES surveys.
```

Comment:
I'm not convinced CES is too narrow, but CSAT is not? They seem very similar so this feels like a weak reason.

Reply:
reworked